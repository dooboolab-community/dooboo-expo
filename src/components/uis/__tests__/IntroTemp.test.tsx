import 'react-native';

import type {ReactElement} from 'react';
import React from 'react';
import type {RenderAPI} from '@testing-library/react-native';
import {render} from '@testing-library/react-native';
import {createTestElement, createTestProps} from '../../../../test/testUtils';

import IntroTemp from '../IntroTemp';

let props: any;
let component: ReactElement;
let testingLib: RenderAPI;

describe('[IntroTemp] render', () => {
  props = createTestProps();

  component = createTestElement(<IntroTemp {...props} />);

  it('renders without crashing', () => {
    testingLib = render(component);

    const baseElement = testingLib.toJSON();

    expect(baseElement).toMatchSnapshot();
    expect(baseElement).toBeTruthy();
  });
});
