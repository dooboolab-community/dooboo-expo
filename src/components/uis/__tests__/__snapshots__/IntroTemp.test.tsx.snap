// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[IntroTemp] render renders without crashing 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "flexDirection": "column",
        "height": "100%",
        "justifyContent": "flex-start",
        "width": "100%",
      },
      undefined,
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#FFFFFF",
          "fontSize": 18,
          "lineHeight": 27,
        },
        {
          "marginTop": 100,
        },
      ]
    }
  />
  <Text
    style={
      [
        {
          "color": "#FFFFFF",
          "fontSize": 18,
          "lineHeight": 27,
        },
        undefined,
      ]
    }
  />
  <Text
    style={
      [
        {
          "color": "#FFFFFF",
          "fontSize": 18,
          "lineHeight": 27,
        },
        undefined,
      ]
    }
  />
</View>
`;
